avahi.py

```diff
@@ -1,26 +1,29 @@
-#!/usr/bin/env python
+# !/usr/bin/env python
```

Comment:
Did this get auto-corrected? 
`#!` is a special case to tell bash what executable to run - does this break that use case?